### DISCUSSION TITLE
food/greens/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/food#2] [General]
# food/greens/

https://shane0.github.io/food/greens/

**Comment by shane0** [comment]
harvest in the morning
https://youtu.be/3HgrPyyuc7A

**Comment by shane0** [comment]
Types
https://youtu.be/b8eDaeWaK1A?si=Va9XRIZJoZalJixo

**Comment by shane0** [comment]
https://youtu.be/3_Wt0O9--34
